homeassistant/components/amcrest/__init__.py

```diff
@@ -5,19 +5,24 @@
 import aiohttp
 import voluptuous as vol
 
+from homeassistant.components.binary_sensor import DOMAIN as BINARY_SENSOR
+from homeassistant.components.camera import (
+    CAMERA_SERVICE_SCHEMA, DOMAIN as CAMERA)
+from homeassistant.components.sensor import DOMAIN as SENSOR
+from homeassistant.components.switch import DOMAIN as SWITCH
 from homeassistant.const import (
-    CONF_NAME, CONF_HOST, CONF_PORT, CONF_USERNAME, CONF_PASSWORD,
-    CONF_BINARY_SENSORS, CONF_SENSORS, CONF_SWITCHES, CONF_SCAN_INTERVAL,
-    HTTP_BASIC_AUTHENTICATION)
+    ATTR_ENTITY_ID, CONF_AUTHENTICATION, CONF_BINARY_SENSORS, CONF_HOST,
+    CONF_NAME, CONF_PASSWORD, CONF_PORT, CONF_SCAN_INTERVAL, CONF_SENSORS,
+    CONF_SWITCHES, CONF_USERNAME, ENTITY_MATCH_ALL, HTTP_BASIC_AUTHENTICATION)
 from homeassistant.helpers import discovery
 import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.service import async_extract_entity_ids
 
 REQUIREMENTS = ['amcrest==1.3.0']
```

Comment:
Your branch still having this means it's quite outdated, please rebase.

Reply:
Yes, saw that. I was wondering if I should remove them, and I saw they already were. This is only that old because it sat around for a week. 😄 I'll rebase again. Thanks!